index.js

```diff
@@ -15,9 +15,11 @@ secondBookmarkButton.addEventListener("click", () => {
 
 const showAnswerButton = document.querySelector("[data-js=show-answer-button]");
 const firstCardAnswer = document.querySelector("[data-js=card-one-answer]");
-
-showAnswerButton.addEventListener("click", () => {
+showAnswerButton.addEventListener("click", (toggleButtonText) => {
```

Comment:
```suggestion
showAnswerButton.addEventListener("click", () => {
```
You don't pass a parameter here, and you also don't not use it, or? So you could remove this.

Reply:
Forgot to remove the togglebuttonText reference, already removed the corresponding function. The console stuff ... sometimes i like to keep these messages to add a little fun. But in this case i forgot as well.